workflow/rules/common.smk

```diff
@@ -3,12 +3,17 @@ __copyright__ = "Copyright 2021, Jonas Almlöf"
 __email__ = "jonas.almlof@scilifelab.uu.se"
 __license__ = "GPL-3"
 
-import pandas as pd
+
+import pandas as pandas
```

Comment:
same as in `prealignment`, `as pandas` is probably not required.

Reply:
done